data/kg/compound_chebi/meta.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: compound_chebi
+description: Knowledgegraph data samples.
+targets:
+    - id: node2_type
+      description: node2_type
+      type: Other
+      units: node2_type
+      names:
+          - noun: node2_type
+    - id: node2_name
+      description: node2_name
+      type: Other
+      units: node2_name
+      names:
+          - noun: node2_name
+    - id: node2_id
+      description: node2_id
+      type: Other
+      units: node2_id
+      names:
+          - noun: node2_id
+identifiers:
+    - id: node1_type
+      description: node1_type
+      type: Other
+    - id: node1_smiles
+      description: node1_smiles
+      type: Other
+    - id: node1_name
+      description: node1_name
+      type: Other
+    - id: node1_id
+      description: node1_id
+      type: Other
+    - id: rel1_type
+      description: rel1_type
+      type: Other
+license: CC BY 4.0
+links:
+    - url: https://crossbar.kansil.org
+      description: original knowledge graph web GUI link
+num_points: 0
+bibtex:
+    - "@article{10.1093/nar/gkab543,\nauthor = {Doğan, Tunca and Atas, Heval and Joshi, Vishal and Atakan, Ahmet and Rifaioglu, Ahmet Sureyya and Nalbat,\
+      \ Esra and Nightingale, Andrew and Saidi, Rabie and Volynkin, Vladimir and Zellner, Hermann and Cetin-Atalay, Rengul and Martin, Maria and Atalay,\
+      \ Volkan},\ntitle = \"{CROssBAR: comprehensive resource of biomedical relations with knowledge graph representations}\",\njournal = {Nucleic Acids\
+      \ Research},\nvolume = {49},\nnumber = {16},\npages = {e96-e96},\nyear = {2021},\nmonth = {06},\nissn = {0305-1048},\ndoi = {10.1093/nar/gkab543},\n\
+      url = {https://doi.org/10.1093/nar/gkab543},\n}"
+templates:
+    - The {node1_type#} SMILES {node1_smiles#} {rel1_type#} {node2_name#}.
```

Comment:
what is here an example of `node1_type#`?

Reply:
Here is the first raw of the cleaned data:
```
node1_type,node1_smiles,node1_name,node1_id,rel1_type,node2_type,node2_name,node2_id
Compound,NCCCCCCN,CHEMBL303004,CHEMBL303004,is a,chebi,"hexane-1,6-diamine",CHEBI:39618
```
so `node1_type` is `Compound` (will casted to lower case in the insertion process)
`rel1_type ` is `is a `
`node2_type` is `chebi `